package.json

```diff
@@ -1,8 +1,12 @@
 {
-  "name": "onepay-sdk",
-  "version": "1.5.9",
-  "description": "Transbank SDK for JS front ends",
-  "main": "lib/onepay-sdk.js",
+  "name": "transbank-onepay-sdk",
+  "version": "1.5.10",
+  "description": "Transbank Onepay SDK",
```

Comment:
I'm not sure this is a good description: If we ever create the nodejs transbank-sdk and we incorporate the backend parts of onepay there, it will be cofusing. I don't think the previous description was great, but I think it was good to mention the frontend (or browser or whatever) part to make it clear it's not the backend SDK :)

Reply:
@leosoto ok, agreed. I've just mentioned in the description.

Reply:
@leosoto Waiting for your updated review 😄

Reply:
https://github.com/TransbankDevelopers/transbank-sdk-js-onepay/pull/50/commits/4599bc2b255168fef0fe30209b51d9c5e269c12b